.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+### Summary
+
+<!-- What does this PR do? -->
+
+<!-- * Closes [YOUR_ISSUE_HERE] -->
+
+### Quality checklist
+
+The documentation added in this PR is:
```

Comment:
Is this checklist meant to be filled out by the person who opens the PR? IMO this checklist would be useful for an editor (or peer editor) to fill out _after_ they’ve reviewed the changes and perhaps iterated over fixes with the author. But I'm skeptical that folks who contribute to docs will have the objectivity (or in some cases skills) they needed to assess how well they've achieved these goals. I also think that some of these goals are not achievable in smaller PRs. For example, if I update one sentence in a topic in a section that's poorly organized, do I need to fix the larger organizational problem? Or do I remove the items that I think are not applicable? It's more likely that folks will just tick off everything in the list and say they're done. Will that make the docs any better? I doubt it. Then it just becomes an exercise in saying that we're making the docs better...without actually making the docs better.

I suggest adding a checklist that is more actionable and specific. Checking the guidelines is a good first item, so I would keep that and maybe also point to some other guidelines, like the topic about writing about the UI. I would also focus on specific tasks like: did the contributor run a spell checker? grammar checker? validate the steps? test the code samples? verify that screenshots and images are accurate, if present? read through the changed pages in the preview? make sure the content appears where it's expected in the TOC?

I'm sure there are some other actions we could recommend without asking PR contributors to say they've achieved goals that I think maybe only 2% of our existing content achieves.

At one company where I worked, we had an Editing for Quality (EFQ) process where writers worked very closely with an editor over the course of several weeks and at the end of that period, they would be assessed on criteria that's very similar to this checklist. I think...we are expecting a lot of PR contributors here.